src/main/resources/org/broadinstitute/dsde/sam/liquibase/changesets/20240806_add_profile_info_table.xml

```diff
@@ -0,0 +1,84 @@
+    <addColumn tableName="SAM_USER_ATTRIBUTES">
+        <column name="first_name" type="VARCHAR">
+            <constraints nullable="true"/>
+        </column>
+    </addColumn>
+    <addColumn tableName="SAM_USER_ATTRIBUTES">
+        <column name="last_name" type="VARCHAR">
+            <constraints nullable="true"/>
+        </column>
+    </addColumn>
+    <addColumn tableName="SAM_USER_ATTRIBUTES">
+        <column name="organization" type="VARCHAR">
+            <constraints nullable="true"/>
+        </column>
+    </addColumn>
+    <addColumn tableName="SAM_USER_ATTRIBUTES">
+        <column name="contact_email" type="VARCHAR">
+            <constraints nullable="true"/>
+        </column>
+    </addColumn>
+    <addColumn tableName="SAM_USER_ATTRIBUTES">
+        <column name="title" type="VARCHAR">
+            <constraints nullable="true"/>
+        </column>
+    </addColumn>
+    <addColumn tableName="SAM_USER_ATTRIBUTES">
+        <column name="department" type="VARCHAR">
+            <constraints nullable="true"/>
+        </column>
+    </addColumn>
+    <addColumn tableName="SAM_USER_ATTRIBUTES">
+        <column name="interest_in_terra" type="VARCHAR ARRAY">
+            <constraints nullable="true"/>
+        </column>
+    </addColumn>
+    <addColumn tableName="SAM_USER_ATTRIBUTES">
+        <column name="program_location_city" type="VARCHAR">
+            <constraints nullable="true"/>
+        </column>
+    </addColumn>
+    <addColumn tableName="SAM_USER_ATTRIBUTES">
+        <column name="program_location_state" type="VARCHAR">
+            <constraints nullable="true"/>
+        </column>
+    </addColumn>
+    <addColumn tableName="SAM_USER_ATTRIBUTES">
+        <column name="program_location_country" type="VARCHAR">
+            <constraints nullable="true"/>
+        </column>
+    </addColumn>
+    <addColumn tableName="SAM_USER_ATTRIBUTES">
+        <column name="research_area" type="VARCHAR ARRAY">
+            <constraints nullable="true"/>
+        </column>
+    </addColumn>
+    <addColumn tableName="SAM_USER_ATTRIBUTES">
+        <column name="additional_attributes" type="JSONB">
+            <constraints nullable="true"/>
+        </column>
+    </addColumn>
+    <addColumn tableName="SAM_USER_ATTRIBUTES">
+        <column name="created_at" type="timestamptz">
+            <constraints nullable="true"/>
+        </column>
+    </addColumn>
+    </changeSet>
+    <changeSet logicalFilePath="dummy" author="kfeldman" id="move_updated_at_column_to_end_of_table">
+<!--        renaming updated_at so I can recreate it as a new column, then transfer existing data -->
+        <renameColumn tableName="SAM_USER_ATTRIBUTES" oldColumnName="updated_at" newColumnName="old_updated_at"/>
+        <addColumn tableName="SAM_USER_ATTRIBUTES">
+            <column name="updated_at" type="timestamptz" valueComputed="old_updated_at">
```

Comment:
yes it will persist those values, I just did some manual testing to confirm. It should just be seeding the db with the values from `old_updated_at` but not creating a linkage between those columns, so when we drop `old_updated_at` the new `updated_at` should persist.